src/module-1/cutStrings/index.js

```diff
@@ -1 +1,14 @@
-export const cutStrings = (arr = []) => {};
+export const cutStrings = (arr = []) => {
+  if (arr.length > 0) {
+    let shortestStr = arr.reduce((a, b) => a.length <= b.length ? a : b);
```

Comment:
```suggestion
    const shortestStr = arr.reduce((a, b) => a.length <= b.length ? a : b);
```  
you actually mustn't look for the shortest string here, but for the smallest length, so you won't need to take length once again in line 7.

Reply:
`reduce` is rather performance-impacting function, try to use forEach for finding the smallest length